src/pages/Examples/TopologyPageMedium/TopologyPageMediumView.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { TopologyColumn } from "../../../components/TopologyColumn/TopologyColumn";
+import { TopologyCardMedium } from "../../../components/TopologyColumn/TopologyCardMedium/TopologyCardMedium";
+import { colorHandler } from "../../../utils/common";
+
+export const TopologyPageMediumView = ({ topology }) => {
```

Comment:
So the size should be an attribute e.g. `size="medium"` - we could create an other component called TopologyCard with this attribute, and then choose one of the individual components

Reply:
I think if we cram a bunch of code (actually three different components) into one, it can be hard to maintain and improve. What do you think about the TopologyCard proxy component, which, depending on the received variant, will render one of TopologyCardCompact, TopologyCardMedium, TopologyCardFull components and pass the rest of the props to them?

Reply:
Thats fine